docs/index.html

```diff
@@ -6,6 +6,7 @@
   <meta property="og:description" content="GPU-friendly Stroke Expansion"/>
   <link href="https://fonts.googleapis.com/css2?family=Material+Icons" rel="stylesheet">
   <link href='https://fonts.googleapis.com/css?family=Nunito:300,400,500,600,700,300italic,400italic,500italic,600italic,700italic' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Inconsolata:300,400,500,600,700,300italic,400italic,500italic,600italic,700italic' rel='stylesheet' type='text/css'>
```

Comment:
You only have to bring in the 400 weight here, we don't use additional weights. Similar optimization can be done for Lato reference.